rpc/cloudagent/network/networkinterface/moc_cloudagent_networkinterface.proto

```diff
@@ -33,6 +38,7 @@ message IpConfiguration {
 	IPAllocationMethod allocation = 6;
 	string gateway = 7 [(sensitive) = true];
 	repeated InboundNatRule inboundNatRules = 8;
+	NetworkType networkType = 9;
```

Comment:
Yeah, this is a read-only property and is being set internally when networkInterface create is called. Currently we are overloading `virtualNetworkName` property to accept LNET during vNIC creation. So, we look at this property and if it has `logicalnetwork/` prefix then IpConfig::networkType is set to `LOGICAL_NETWORK` otherwise `VIRTUAL_NETWORK`. `networkType` is used internally to determine which provider to use - vnet or lnet.

I will add a comment if that is helpful.

Reply:
Resolved - Added comment to indicate read-only.